test/infrastructure/kubemark/config/samples/infrastructure_v1alpha3_kubemarkmachine.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: infrastructure.cluster.x-k8s.io/v1alpha4
```

Comment:
The whole `samples` directory can likely be deleted, I believe we already do that for the other generated content as well.